pages/1.12/release-notes/1.12.4/index.md

```diff
@@ -0,0 +1,217 @@
+
+
+
+## Command-line interface (CLI)
+- Provides improved clean-up operations to eliminate issues caused by unused volumes for removed comtainers (DCOS_OSS-1502). 
+
+    Previously, there have been issues caused by orphan volumes that were not remove when their associated containers were removed. In general, you can run  `docker prune` commands to avoid issues with unused volumes. However, with this release, you can also use the `docker-gc` command to remove unused volumes associated with removed containers.
+
+## Exhibitor
+- Changes the security setting for the configuration parameters `aws_secret_access_key` and `exhibitor_azure_account_key` so that these values are not visible in the `user.config.yaml` file on cluster nodes (DCOS-51751). 
+
+    These configuration parameters are used for Exhibitor. They are now marked as secret. The values set for these configuration parameters can only be viewed in the `user.config.full.yaml` file. The `user.config.full.yaml` file has stricter read permissions than the `user.config.yaml` file and is not included in DC/OS diagnostics bundles.
+
+## GUI
+- Includes updates to the DC/OS frontend GUI to improve the user experience (COPS-4796, COPS-4804, COPS-4857, DCOS-53836, DCOS-54034, DCOS-54039, DCOS-54041).
+
+## Installation
+ - Corrects the output returned when running the `dcos_generate_config.sh` or `dcos_generate_config.ee.sh` script with the `--validate-config` option so that it doesn’t display warning or error messages about missing deprecated configuration settings such as `ssh_user` and `ssh_key_path` (COPS-4282, DCOS_OSS-4613, DCOS_OSS-5152).
+
+## Marathon
+- Introduces a watcher loop process to monitor and, if necessary, re-register the Marathon leader after reelection (COPS-3554).<!--Also in previous RN, 1.12.4-->
+
+- Fixes an issue that prevented services managed by Marathon from restarting *COPS-3593, DCOS_OSS-4193).
+
+    In previous releases, you might have services that are managed by Marathon unable to restart if the container crashes or under certain DNS failure conditions. For example, restarting services might fail if the first ZooKeeper node or first DC/OS master is unreachable.
+
+    Because this problem affects high availability for Marathon, a workaround (ping zk-1) was introduced for DC/OS 1.11.5 and 1.11.6 to address the issue. In this release, the underlying issue is resolved and you can safely remove the workaround if you have it deployed. For background information about the issue and the steps to remove the workaround, see [Removing the patch for Marathon failing to start if the first DC/OS is not available](https://mesosphere-community.force.com/s/article/Critical-Issue-Marathon-MSPH-2018-0004). <!--Also in previous RN, 1.12.4-->
+
+## Mesos platform and containerization
```

Comment:
Can you please include the newly added RN from https://jira.mesosphere.com/browse/COPS-4703 as well? Thanks.

Reply:
Added.